packages/lesswrong/components/users/UsersMenu.jsx

```diff
@@ -52,6 +52,7 @@ class UsersMenu extends PureComponent {
         <Button onClick={this.handleClick}>
           <span className={classes.userButton} style={{ color: color }}>
             {Users.getDisplayName(currentUser)}
+            {currentUser && currentUser.karma>0 && ` (${currentUser.karma})`}
```

Comment:
I would suggest putting this in a div instead of having a hard-coded space before the first parenthesis.